### PR TITLE
[LIB-567] Add support for version 1.1 of the API

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

File: test/integration/scheduleTracesTest.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ describe('ScheduleTrace', function() {
       return connection.Schedule.please().create({
         instanceName,
         label: instanceName,
-        codebox: codeBox.id,
+        script: codeBox.id,
```
Comment (dancio) [inline test/integration/scheduleTracesTest.js:35]: `codeBox.id` ? :P

**Comment by dancio** [comment]
when u want to change everything value of integration tests is clearly visible :P, lgtm

File: test/integration/scheduleTracesTest.js (hunk 1 above)
Comment (jakubbilko) [inline test/integration/scheduleTracesTest.js:35]: Well, temporarily yes... since we're changing the names in another task ;)

Comment (dancio) [inline test/integration/scheduleTracesTest.js:35]: :+1:

**Pull request merged**